src/database/migrations/2024_04_04_073207_create_profiles_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('profiles', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('user_id')->constrained()->onDelete('cascade');
+            $table->string('first_name');
+            $table->string('last_name');
+            $table->string('middle_name')->nullable();
+            $table->date('birth_date');
+            $table->string('address');
+            $table->timestamps();
+            $table->timestamp('email_verified_at')->nullable();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::dropIfExists('profiles');
+    }
+};
```

Comment:
trail spacing!

If you see someting like this in your PR just add breakline

![image](https://github.com/xReddskie/microblog-ojt2/assets/115203179/65bb1ebb-c2d4-4ea6-9d8b-c39fe43abe8d)